Documentation/Storage-Configuration/Object-Storage-RGW/object-storage.md

```diff
@@ -82,19 +95,16 @@ spec:
       - ip: 192.168.39.182
 ```
 
-You can use the existing `object-external.yaml` file.
-When ready the ceph-object-controller will output a message in the Operator log similar to this one:
+Use the existing `object-external.yaml` file. Even though multiple endpoints can be specified, it is recommend to use only one endpoint. This endpoint is randomly added to `configmap` of OBC and secret of the `cephobjectstoreuser`. Rook never guarantees the randomly picked endpoint is a working one or not.
+If there are multiple endpoints, please add load balancer in front of them and use the load balancer endpoint in the `externalRgwEndpoints` list.
```

Comment:
This is true now but will need updated when you finish changes to have Rook perform basic HTTP checks to see if the endpoints are active, right?

Reply:
WYes thats true, this aligns with current code in Rook